addons/mail/static/src/discuss/core/common/discuss_core_common_service.js

```diff
@@ -129,6 +129,11 @@ export class DiscussCoreCommon {
                     }
                 }
             });
+            this.busService.subscribe("mail.cannedReponse/delete", (datas) => {
+                for (const data of datas) {
+                    this.store.CannedResponse.get(data).delete();
+                }
+            });
```

Comment:
should have generic `mail.record/delete` with model name as key instead

also data has no plural